mmcls/models/classifiers/image.py

```diff
@@ -33,6 +33,15 @@ def __init__(self,
             if augments_cfg is not None:
                 self.augments = Augments(augments_cfg)
 
+    def forward_dummy(self, img):
+        """Used for computing network flops.
+
+        See `mmclassificaiton/tools/analysis_tools/get_flops.py`
+        """
+        x = self.extract_feat(img)
+        outs = self.head.simple_test(x, post_process=False)
```

Comment:
Should the `forward_dummy` includes `head.simple_test`?
For the classification tasks, we may use different datasets to do pre-training. If the `forward_dummy` includes `head.simple_test`, the flops value of the same model trained on `ImageNet-1k` and `ImageNet-21k` will be different. It's a little confusing.

Reply:
Sounds good.